notebook/agentchat_agentoptimizer.ipynb

```diff
@@ -2,7 +2,12 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "front_matter": {
+     "description": "AgentOptimizer is able to prompt LLMs to iteratively optimize function/skills of AutoGen agents according to the historical conversation and performance.",
+     "tags": ["self-improving"]
```

Comment:
Shall we add "optimization"? "learning"? "function call"?
btw maybe we should change "function call" to "tool/function"

Reply:
Yes. I think maybe we can use "optimization" as one new tag to include all future features related to LLM optimization capability.